src/bootstrap/bootstrap.py

```diff
@@ -931,7 +942,7 @@ def build_triple(self):
         so use `self.build` where possible.
         """
         config = self.get_toml('build')
-        return config or default_build_triple(self.verbose)
+        return config or default_build_triple(self.verbose, self.build_dir)
```

Comment:
Passing the self directly would make more sense IMO

Reply:
I think it's like this because `configure.py` also uses `default_build_triple`, and there's no `self` for it to pass there